8vim/src/main/res/drawable/custom_seekbar_thumb.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
There needs to be consistency across a page and the whole application
You can not design a custom seekbar thumb for a single seekbar on a single activity.